src/components/search.js

```diff
@@ -1,8 +1,32 @@
-import React from 'react';
-import SearchResults from './search_results';
-import { useHistory } from "react-router-dom";
+import React, { useState } from 'react';
+import cocktailStore from '../stores/cocktail_store';
+import { GET_COCKTAILS, PERFORM_SEARCH } from '../constants';
```

Comment:
Not certain we're using the PERFORM_SEARCH constant here (?). I see that this is searching, but it seems to mainly be using the GET_COCKTAILS constant

Reply:
Cool, good catch. yeah I don't think it's a used one